res/values/colors.xml

```diff
@@ -168,4 +168,7 @@
     <color name="palette_list_dark_mode_color_blue">#8AB4F8</color> <!-- Material Blue 300 -->
     <color name="palette_list_dark_mode_color_purple">#c58af9</color> <!-- Material Purple 300 -->
 
-</resources>
\ No newline at end of file
+    <!-- YASP Night Mode Color. -->  
+    <color name="ic_yaap_settings_background">@android:color/white</color>
```

Comment:
Move this into a separate yaap_colors.xml file? Don't wanna mess with aosp xml files.